src/main/java/Duke.java

```diff
@@ -10,9 +13,19 @@ public static void main(String[] args) {
         System.out.println("Hello! I'm Duke");
         System.out.println("What can I do for you?");
         System.out.println("_______________________________________________________");
-        System.out.println("Bye. Hope to see you again soon!");
-        for (String arg : args) {
-            System.out.println("_______________________________________________________\n" + arg + "\n" + "_______________________________________________________\n");
+        System.out.println("Your instructions, my Liege.");
+        while (true) {
+            Scanner myObj = new Scanner(System.in);
+            String userInput = myObj.nextLine();
+            if (Objects.equals(userInput, "bye")) {
+                System.out.println("""
+                        _______________________________________________________
+                        Bye! Hope to see you again soon.
+                        _______________________________________________________
+                        """);
+                break;
+            }
+            System.out.println("_______________________________________________________\n" + userInput + "\n" + "_______________________________________________________\n");
```

Comment:
I think you can break up this line into shorter lines! Not sure if it's above the 120-character hard limit but looks significantly longer than the rest of the code